docs/tutorials/deploy_openfaas_workload.md

```diff
@@ -0,0 +1,291 @@
+
+We could deploy it by:
+
+```shell script
+$ kubectl apply -f examples/openfaasapp.yaml
+componentschematic.core.oam.dev/openfaas configured
+applicationconfiguration.core.oam.dev/openfaas created
+``` 
+
+Then you could find functions have been created:
+
+```shell script
+$ kubectl get functions
+NAME       AGE
+nodeinfo   27s
+```
+
+You could also visit the function deployed:
+
+```shell script
+$ curl http://127.0.0.1:31112/function/nodeinfo
+Hostname: nodeinfo-86b679f76f-sgkbb
+
+Platform: linux
+Arch: x64
+CPU count: 2
+Uptime: 796748
+```
+
+## Apply trait
+
+Manual Scaler trait could also be applied to OpenFaaS workload like below:
+
+```yaml
+apiVersion: core.oam.dev/v1alpha1
+kind: ApplicationConfiguration
+metadata:
+  name: openfaas
+spec:
+  components:
+    - componentName: openfaas
+      instanceName: nodeinfo
+      parameterValues:
+        - name: image
+          value: functions/nodeinfo
+        - name: handler
+          value: "node main.js"
+        - name: write_debug
+          value: "false"
+      traits:
+        - name: manual-scaler
+          parameterValues:
+            - name: replicaCount
+              value: 2
+```
+
+After change the Application Configuration yaml file, apply it.
+
+```
+$ kubeclt apply -f examples/openfaasapp.yaml
+componentschematic.core.oam.dev/openfaas unchanged
+applicationconfiguration.core.oam.dev/openfaas configured
+```
+
+Then we could see the replica of the function has changed.
+
+```shell script
+$ kubectl get functions nodeinfo -o yaml
+apiVersion: openfaas.com/v1alpha2
+kind: Function
+metadata:
+  name: nodeinfo
+  namespace: default
+spec:
+  environment:
+    write_debug: "false"
+  handler: node main.js
+  image: functions/nodeinfo
+  name: nodeinfo
+  replicas: 2
+```
+
+If you want to change the replicas to `3`, just change the application configuration yaml.
+
+```yaml
+apiVersion: core.oam.dev/v1alpha1
+kind: ApplicationConfiguration
+metadata:
+  name: openfaas
+spec:
+  components:
+    - componentName: openfaas
+      instanceName: nodeinfo
+      parameterValues:
+        - name: image
+          value: functions/nodeinfo
+        - name: handler
+          value: "node main.js"
+        - name: write_debug
+          value: "false"
+      traits:
+        - name: manual-scaler
+          parameterValues:
+            - name: replicaCount
+              value: 3
+```
+
+Apply it again.
+
+```
+$ kubeclt apply -f examples/openfaasapp.yaml
+componentschematic.core.oam.dev/openfaas unchanged
+applicationconfiguration.core.oam.dev/openfaas configured
+```
+
+Then you could see the `replicas` field has been changed.
+
+```shell script
+$ kubectl get functions nodeinfo -o yaml
+apiVersion: openfaas.com/v1alpha2
+kind: Function
+metadata:
+  name: nodeinfo
+  namespace: default
+spec:
+  environment:
+    write_debug: "false"
+  handler: node main.js
+  image: functions/nodeinfo
+  name: nodeinfo
+  replicas: 3
```

Comment:
We will be deprecating this value please use proper scaling labels instead. Set min scale to control replicas 

https://docs.openfaas.com/architecture/autoscaling/

Reply:
sure, thanks!